src/Parser/Anime/AnimeParser.php

```diff
@@ -664,6 +664,24 @@ public function getExternalLinks(): array
             });
     }
 
+    /**
+     * @return array
+     */
+    public function getStreamingLinks(): array
+    {
+        $links = $this->crawler
+            ->filterXPath('//*[@id="content"]/table/tr/td[1]/div/div[contains(@class, "broadcast")]//div[contains(@class, "broadcast")]');
+
+        if (!$links->count()) {
+            return [];
+        }
+
+        return $links->filterXPath('//a')
+            ->each(function (Crawler  $c) {
+                return (new UrlParser($c))->getModel();
+            });
```

Comment:
```suggestion
        return $links->count() > 0 ? $links->filterXPath('//a')
            ->each(function (Crawler  $c) {
                return (new UrlParser($c))->getModel();
            }) : [];
```
Personally I prefer single return points in function bodies, and I'll try to strife for those kind of solutions where possible. This one can be simplefied to a single return via an inline if statement.

Reply:
Eitherway's fine IMO. I'm just keeping things consistent for where multiple early returns are being carried out:
https://github.com/jikan-me/jikan/blob/972ffdb947c75a901836cb68b7845b808df57954/src/Parser/Reviews/AnimeReviewParser.php#L93

Reply:
Then leave it, it's fine.